theseus_gui/src/components/ui/Instance.vue

```diff
@@ -0,0 +1,144 @@
+<script setup>
+import { RouterLink } from 'vue-router'
+import { Card } from 'omorphia'
+
+const props = defineProps({
+  display: String,
+  instance: Object,
+})
+</script>
+
+<template>
+  <div>
+    <RouterLink v-if="display === 'list'" class="instance-list-item" :to="`${props.instance.id}`">{{
+      props.instance.name
+    }}</RouterLink>
+    <div class="instance-gallery-item" v-else-if="display === 'gallery'">
+      <p>{{ props.instance.version }}</p>
+      <p>{{ props.instance.name }}</p>
+    </div>
+    <Card class="instance-card-item" v-else-if="display === 'card'">
+      <img src="https://cdn.modrinth.com/data/AANobbMI/icon.png" alt="Trending mod card" />
+      <div class="project-info">
+        <p class="title">{{ props.instance.name }}</p>
+        <p class="description">{{ props.instance.description }}</p>
+      </div>
+    </Card>
```

Comment:
The text could be hard to see depending on the icon. Maybe we should move to a spotify-like instance card design?